bundles/org.openhab.io.homekit/README.md

```diff
@@ -994,3 +994,32 @@ You can verify this with [Discovery DNS iOS app](https://apps.apple.com/us/app/d
   - if sf is equal 1, openHAB is accepting pairing from new iOS device. 
   - if sf is equal 0 (as on screenshot), openHAB is already paired and does not accept any new pairing request. you can reset pairing using `openhab:homekit clearPairings` command in karaf console.
 - if you see openHAB bridge and sf is equal 1 but you dont see openHAB in home app, probably you home app still think it is already paired with openHAB. remove your home from home app and restart iOS device.
+
+### Re-adding the OpenHAB HomeKit bridge reports that a bridge is already added, even though it has clearly been removed.
+
+There are various reasons this may happen. Try the following:
+
+* In [openhab-cli](https://www.openhab.org/docs/administration/console.html), run `openhab:homekit clearPairing`. Reboot openHAB. Reboot your iOS device. Try again.
+* In the Homekit settings, change the port, setupId, and pin. Save the settings, then re-open the settings so as to refresh the QR code. Re-add the device.
+* Remove Homekit state in `/var/lib/openhab/jsondb/homekit.json`, and homekit config in `/var/lib/openhab/config/org/openhab/homekit.config`. Restart openHAB. Reboot iPhone. Try again.
```

Comment:
I don't think we should be recommending to do either of these steps. After https://github.com/openhab/openhab-addons/pull/14016, I was able to rapid fire pair, depair-on-iOS, clearPairings, and re-pair multiple times in a row with no issues. The important thing is that clearPairings will be absolutely necessary if you have multiple users accessing your Home. iOS will quickly add the additional users shortly after pairing, but only remove the user requesting to remove the bridge when un-pairing, so you have to clear out the other users.

I'd rather focus our time on figuring out how we can get "Clear Pairings" (and "Prune Dummy Accessories") as a button in the config, similar to "Heal ZWave Network" or whatever it is is a button on the thing config for a Zwave network.

Reply:
I ran in to issues with 3.4.0 where clearPairings was insufficient, and these were effectively the steps I followed.

It would be great to get clearPairings working properly.

Maybe we keep these steps but try and make it so the first step is sufficient?